docs/core/compatibility/wcf-client/6.0/duplex-synchronization-context.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Breaking change: Duplex contract captures synchronization context"
+description: Learn about the breaking change in WCF Client 6.0 where the duplex contract now captures synchronization context when it's opened.
```

Comment:
description: Learn about the breaking change in WCF Client 6.0 where the DuplexChannelFactory now captures the synchronization context when it's opened.